apps/src/lib/kits/maker/ui/SetupInstructions.jsx

```diff
@@ -82,16 +91,112 @@ class ConnectionInstructions extends React.Component {
           <li>{applabI18n.makerSetupWebSerialConnectToComputer()}</li>
           <li>{applabI18n.makerSetupWebSerialConnectToBoardButton()}</li>
           <li>
-            <SafeMarkdown
-              markdown={applabI18n.makerSetupWebSerialWindowConnect()}
-            />
+            {applabI18n.makerSetupWebSerialWindowConnect()}
+            <ul className={styles.troubleshootList}>
+              <li>
+                <SafeMarkdown
+                  markdown={applabI18n.makerSetupNoCompatibleDevices({
+                    installInstructionsId: styles.installInstructionsId,
+                  })}
+                />
+              </li>
+            </ul>
           </li>
           <li>{applabI18n.makerSetupWebSerialSuccessfulConnection()}</li>
         </ol>
         <SafeMarkdown
           markdown={applabI18n.makerSetupWebSerialSupportArticle()}
+          openExternalLinksInNewTab={true}
         />
         {connectionState}
+        <CPExpressInstallInstructions />
```

Comment:
It seems to me that these 'install' instructions are separate from the 'connection' instructions. What do you think of moving `CPExpressInstallInstructions` above to [line 31](https://github.com/code-dot-org/code-dot-org/blob/aebd9d2979ad70a665d2474a776ef236ad57e2c1/apps/src/lib/kits/maker/ui/SetupInstructions.jsx#L31).

Reply:
That is where I originally had it, but I moved it here because if you load the page from somewhere the doesn't support WebSerial (!shouldUseWebSerial()) then we don't show the ConnectionInstructions (return null on line 77) so I thought it'd be weird to show these instructions in that case.

Maybe it would be clearer to pull that logic up into the SetupInstructions

Reply:
Yeah - I agree it would make sense to pull logic into setup instructions and instead of returning `null` we could return an alternative message that states that the device/computer needs to support WebSerial in order to be able to use the Code.org Maker Toolkit.

Reply:
Discussed offline that returning an alternative message is outside scope of this task and could be a follow-up task.

Reply:
Issue created to address the alternative message https://codedotorg.atlassian.net/browse/LABS-508